tendrl/integrations/ceph/objects/definition/ceph.yaml

```diff
@@ -18,3 +18,76 @@ namespace.integrations.ceph:
       type: Create
       uuid: 2f94a48a-05d7-408c-b400-e27827f4edcd
       version: 1
+  objects:
+    Osd:
+      attrs:
+        id:
+          help: Id of the OSD
+          type: int
+        uuid:
+          help: UUID
+          type: String
+        hostname:
+          help: Name of the host
+          type: String
+        public_addr:
+          help: Public Address
+          type: String
+        cluster_addr:
+          help: Cluster Address
+          type: String
+        device_path:
+          help: device path
+          type: String
+        heartbeat_front_addr:
+          help: Heartbeat front address
+          type: String
+        heartbeat_back_addr:
+          help: Heartbeat back address
+          type: String
+        down_at:
+          help: Down at
+          type: int
+        up_from:
+          help: Up from
+          type: int
+        lost_at:
+          help: Lost at
+          type: int
+        osd_up:
+          help: OSD up flag
+          type: int
+        osd_in:
+          help: OSD int flag
+          type: int
+        up_thru:
+          help: Up through
+          type: int
+        weight:
+          help: Weight
+          type: String
+        primary_affinity:
+          help: Primary affinity
+          type: String
+        state:
+          help: State
+          type: String[]
+        last_clean_begin:
+          help: Last clean begin
+          type: int
+        last_clean_end:
+          help: Last clean end
+          type: int
+        total:
+          help: total size of osd
+          type: int
+        used:
+          help: used size
+          type: int
+        used_pcnt:
+          help: used percent
+          type: String
+      help: "Osd"
+      enabled: true
+      value: clusters/$TendrlContext.integration_id/Osds/$Osd.uuid
```

Comment:
I had asked you to remove uuid, there's no rush, read comments and then implement